apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/test/java/com/expediagroup/apiary/extensions/events/metastore/kafka/listener/KafkaMetaStoreEventListenerTest.java

```diff
@@ -45,23 +42,20 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.invocation.InvocationOnMock;
 import org.mockito.junit.MockitoJUnitRunner;
-import org.mockito.stubbing.Answer;
-
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableAddPartitionEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableAlterPartitionEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableAlterTableEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableCreateTableEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableDropPartitionEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableDropTableEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableInsertEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableListenerEvent;
-import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableListenerEventFactory;
-import com.expediagroup.apiary.extensions.events.metastore.common.io.MetaStoreEventSerDe;
-import com.expediagroup.apiary.extensions.events.metastore.common.messaging.Message;
-import com.expediagroup.apiary.extensions.events.metastore.common.messaging.MessageTask;
-import com.expediagroup.apiary.extensions.events.metastore.common.messaging.MessageTaskFactory;
+
+import com.expediagroup.apiary.extensions.events.metastore.kafka.common.event.SerializableAddPartitionEvent;
```

Comment:
When we move these out to their own module let's replace "Serializable" with "Apiary".